<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Display2DDialog</class>
 <widget class="QDialog" name="Display2DDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1107</width>
    <height>932</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCustomPlot" name="customPlot_2Dmotion" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>771</width>
     <height>601</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="status_text">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>780</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>status</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_north">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>470</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>northlabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mousepos">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>790</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="customPlot_6DOF" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>771</width>
     <height>601</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>../../qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
